pubspec.yaml

```diff
@@ -58,8 +58,8 @@ flutter:
   uses-material-design: true
 
   assets:
-    - assets/bluegradient.png
-    - assets/loginpageimage.png
```

Comment:
Are we using the same image for register and login? Just want to make sure these lines were not deleted by mistake. And if not, rename it, since it's not only used in register. Also is this gradient the same in the entire app? Then it definitely should not have "register" in its name.